test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -932,6 +989,7 @@ def create_kwargs():
             create_kwargs,
             load_checkpoint_during_init=self.load_checkpoint_during_init,
             export_within_fake_mode=self.export_within_fake_mode,
+            dynamic_shapes=self.dynamic_shapes,
         )
 
     def test_large_scale_exporter_with_tiny_gpt2(self):
```

Comment:
> 2023-08-03T01:22:00.0224061Z FAILED [2.5878s] onnx/test_fx_to_onnx_with_onnxruntime.py::TestFxToOnnxFakeTensorWithOnnxRuntime_op_level_debug_True_dynamic_shapes_True_load_checkpoint_during_init_True_export_within_fake_mode_True::test_large_scale_exporter_with_tiny_gpt2 - AssertionError: Dynamic shape check failed for graph inputs

This is the one failing in CI. Need more info. Try adding `verbose=True` and grab the sarif log. Each 'pass' diagnostic would show the graph input shape which may tell us at which step they became static.

Reply:
BTW the CI log has too many garbage logs, I'm fixing that!